0065-swiftui-and-state-management-pt1/StateManagement.playground/Contents.swift

```diff
@@ -29,18 +29,12 @@ private func ordinal(_ n: Int) -> String {
 
 import Combine
 
-class AppState: BindableObject {
-  var count = 0 {
-    willSet {
-      self.willChange.send()
-    }
-  }
-
-  var willChange = PassthroughSubject<Void, Never>()
+class AppState: ObservableObject {
+  @Published var count = 0
```

Comment:
things got much simpler for creating models that notify when any of their fields are changed. now you just need to mark each as `@Published`. this must mean property wrappers have access to `self` now, so this is pretty close to what we tried sketching out a few weeks ago.